dev/.rat-excludes

```diff
@@ -98,3 +98,4 @@ LZ4BlockInputStream.java
 spark-deps-.*
 .*csv
 .*tsv
+org.apache.spark.scheduler.ExternalClusterManager
```

Comment:
The version of `ExternalClusterManager` in the PR has a license header in it. Can you remove this and try ?